routes/currentUser.js

```diff
@@ -0,0 +1,15 @@
+const currentUserRouter = require('express').Router();
+const asyncHandler = require('express-async-handler');
+const requireCurrentUser = require('../middlewares/requireCurrentUser');
+
+currentUserRouter.get(
+  '/',
+  requireCurrentUser,
+  asyncHandler(async (req, res) => {
+    console.log(req.currentUser);
+
+    res.json(req.currentUser);
```

Comment:
Probably need to wrap it in a getSafeAttributes() function like Pierre did.
Also, need to create this function in the model 😉 

```suggestion
    res.json(User.getSafeAttributes(req.currentUser));
```